content/en/tracing/guide/trace_sampling_and_storage.md

```diff
@@ -377,6 +377,8 @@ Individual traces are stored for up to 6 months. To determine how long a particu
 | 6 days                 |               10% |
 | 6 months               |                1% |
 
+**Note**: Datadog does not sample Synthetics traces, we store all of the received traces for 6 hours, and the stated percent of the traces over time.
```

Comment:
```suggestion
**Note**: Datadog does not sample Synthetics APM traces. All received traces are stored for 6 hours, and the stated percent of traces over time.
```

Reply:
I don't understand the part `and the stated percent of the traces over time.`.. i may be lost in translation here, but what are you trying to say?

Reply:
I am trying to say that it's 25% of all the traces for a current day, 10% of all the traces for 6 days, and 1% for 6 months. Definitely open to suggestions for how best to say this. Maybe it needs its own section?